File: Fahmi_FP.py
#Tugas 1
a = lambda start, stop: list(map(lambda x: x, range(start, stop + 1)))
print(a(2, 10))

#Tugas 2
y = lambda num: "FizzBuzz" if num % 3 == 0 and num % 5 == 0 else "Fizz" if num % 3 == 0 else "Buzz" if num % 5 == 0 else num
x = list(map(y, range(2, 10)))
print(x)

#Tugas 3
lambdaa = lambda l: list(map(lambda i: l[i] + l[i + 1], range(len(l) - 1)))
lis = [1, 2, 3, 4, 5]
p = lambdaa(lis)
print(p)

File: Fahmi_FP.dart
void main() {
  var a = (int start, int stop) =>
      List.generate(stop - start + 1, (i) => i + start);
  print(a(2, 10));

  var y = (int num) => num % 3 == 0 && num % 5 == 0
      ? "FizzBuzz"
      : num % 3 == 0
          ? "Fizz"
          : num % 5 == 0
              ? "Buzz"
              : num;
  var x = List.generate(8, (i) => y(i + 2));
  print(x);

  var lambdaa =
      (List<int> l) => List.generate(l.length - 1, (i) => l[i] + l[i + 1]);
  var lis = [1, 2, 3, 4, 5];
  var p = lambdaa(lis);
  print(p);
}
